GS01SUMO/GS01SUMO_R00003_ingest.csv

```diff
@@ -0,0 +1,79 @@
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl16/optaa2/*.optaa2.log,GS01SUMO-RID16-01-OPTAAD000,recovered_host,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl16/flort2/*.flort2.log,GS01SUMO-RID16-02-FLORTD000,recovered_host,Available,
+#mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl16/ctdbp/*ctdbp.log,GS01SUMO-RID16-03-CTDBPF000,recovered_host,Not Expected,No data available
+mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl16/CTDBP_sn*/*.hex,GS01SUMO-RID16-03-CTDBPF000,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl16/velpt/*.velpt.log,GS01SUMO-RID16-04-VELPTA000,recovered_host,Available,
+#mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl16/VELPT_sn*/*.aqd,GS01SUMO-RID16-04-VELPTA000,recovered_inst,Not Expected,Instrument not logging internally
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl16/pco2w/*.pco2w.log,GS01SUMO-RID16-05-PCO2WB000,recovered_host,Available,
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl16/PCO2W_snC0050/SAMI*.txt,GS01SUMO-RID16-05-PCO2WB000,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl16/dosta2/*dosta2.log,GS01SUMO-RID16-06-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.nutnr_b.dcl_full.nutnr_b_dcl_full_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl16/nutnr2/*.nutnr2.log,GS01SUMO-RID16-07-NUTNRB000,recovered_host,Available,
+#mi.dataset.driver.nutnr_b.nutnr_b_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl16/NUTNR_sn*/*.DAT,GS01SUMO-RID16-07-NUTNRB000,recovered_inst,Not Expected,No data available
+mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl16/spkir2/*.spkir2.log,GS01SUMO-RID16-08-SPKIRB000,recovered_host,Available,
+mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/cg_data/dcl11/imm/adcp/adcp*.DAT,GS01SUMO-RII11-02-ADCPSN010,recovered_host,Available,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl11/imm/ADCP_sn*/*.000,GS01SUMO-RII11-02-ADCPSN010,recovered_inst,Available,
+#mi.dataset.driver.ctdbp_p.ctdbp_p_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00003/instruments/dcl11/imm/CTDBP_sn16-50159/SBE*.hex,GS01SUMO-RII11-02-CTDBPP031,recovered_inst,Not Expected,No data available

```

Comment:
Interesting that this has everything but the .hex file.  I wonder if we have that and can find it somewhere... on the Data Drive?